lib/pkcs7.js

```diff
@@ -342,16 +342,20 @@ p7.createSignedData = function() {
         if('content' in msg) {
           var content;
           if(msg.content instanceof forge.util.ByteBuffer) {
-            content = msg.content.bytes();
+            content = asn1.create(asn1.Class.UNIVERSAL, asn1.Type.OCTETSTRING, false,
```

Comment:
Nevermind -- I see that the OCTETSTRING below was, in effect, moved up here (this is not a new one that wraps it).